### ISSUE TITLE
Autofill QuickType - show database name

### ISSUE THREAD

**Issue by georgesnow** [strongbox-password-safe/Strongbox#600]
For both platforms would it be possible to add for display format 

Database name (username)

Or

Database name (title)

Where database name is the file name of the database. I would love to have all 3 but there isn't enough room 😕
Thanks

**Comment by strongbox-mark** [comment]
We could do the Nickname of the database, which I think would make more sense than the filename. How about that?

**Comment by georgesnow** [comment]
Sure anything would be better. I have 2 databases with logins that overlap. So having a way differentiate would be super helpful

**Comment by strongbox-mark** [comment]
This is available for you now as of 1.55.1 and 1.16.13 on the App Store now... Please reopen if it's not as you expect.

**Comment by georgesnow** [comment]
So oddity has occurred. On macOS I enabled 1 database appeared to work.

I did the same on iOS worked initially, but when I turned it on for the other 2 databases the QuickType doesn't appear to work reliably. And I can only see 1 entry even though I have a match in each of the respective databases that previously worked. 

Changing it back to Title (username) restores QuickType for the all databases. 

It appears to happen with any of the new string formats that have [] for the database name. Not sure if it's string length or character issue with the brackets.

🤷‍♂️

**Comment by georgesnow** [comment]
@strongbox-mark i tried on my other Mac same deal. Do you want me to create a new ticket or reopen this one?

**Comment by strongbox-mark** [comment]
Hey @georgesnow - I think this is just an artifact of the existing issue with having multiple databases enabled for QuickType.

The QuickType system only allows addition of entries or a full delete. You can't update existing entries. This means for a single database things are fine, we can delete and re-add the entries. When you have multiple databases, we can't delete existing QuickType entries, so we choose the least worst option of appending entries. 

So, if you want to confirm if this is really a bug, or just a fact of having multiple databases, you would need to clear the AutoFill QuickType entries after having configured all your databases to the desired format. Then open each database so that Strongbox populates the QuickType format correctly.

So, make sure the prompts are configured as you like them, then to clear the QuickType database, go to Strongbox AutoFill preferences and toggle the QuickType Enabled switch off then back on.

**Comment by unicorn855** [comment]
Hi @georgesnow :) I literally stumbled upon that quirk/ iOS API shortcoming yesterday, after some configuring of the QuickType settings for each database, I toggled them off for both databases in question, then back on again. I then also restarted my iPad just to be sure and it started working fine after that, so I do believe that - as @strongbox-mark said - it is an artifact relating to https://strongboxsafe.com/support/#reamaze#0#/kb/autofill/there-are-duplicate-old-or-multiple-copies-of-items-or-entries-in-the-autofill-suggestions-quicktype :) Anyway, it's working fine now with the two databases and autofill/QuickType (even the suggestions showed up properly again, but only after I unlocked the second database)

**Comment by strongbox-mark** [comment]
Thanks @unicorn855 - I'm going to close this one out.
